### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by SebastienValla**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,32 @@
-# massa-as-sdk
+# Massa-as-sdk
 
-Massa Assemblyscript SDK
+Collection of tools, objects and functions, that are specific in usage only for Massa smart contracts in AssemblyScript 
+
+Complete doc of all functions and object can be found here : 
+
+- [`massa-as-sdk documentation`](https://massa.net)
```
Comment (Thykof) [inline README.md:7]:
```suggestion
- [`massa-as-sdk documentation`](http://as-sdk.docs.massa.net)
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,32 @@
...
+```sh
+npm run fmt
+```
+
+## To test
+```sh
+npm run test
+```
+
+#Example
```
Comment (Thykof) [inline README.md:23]:
```suggestion
# Example
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,32 @@
...
+
+For instance here is a smart contract example generating an event "Hello World" on the blockchain 
+
+import { generateEvent } from '@massalabs/massa-as-sdk';
+```sh
```
Comment (Thykof) [inline README.md:28]:
```suggestion
```sh
import { generateEvent } from '@massalabs/massa-as-sdk';
```

**Review by Thykof — CHANGES_REQUESTED** [review]
great!

**Review by Thykof — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,3 +1,37 @@
-# massa-as-sdk
+# Massa-as-sdk
```
Comment (0xMazout) [inline README.md:2]:
```suggestion

_Massa Assemblyscript SDK_

```

File: README.md [hunk 5]
```diff
@@ -1,3 +1,37 @@
...
 
-Massa Assemblyscript SDK
+Collection of tools, objects and functions, that are specific in usage only for Massa smart contracts in AssemblyScript 
+
+Complete doc of all functions and object can be found here : 
+
+- [`massa-as-sdk documentation`](http://as-sdk.docs.massa.net)
```
Comment (0xMazout) [inline README.md:7]:
```suggestion
- [`massa-as-sdk documentation`](https://as-sdk.docs.massa.net)
```

**Review by 0xMazout — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -1,3 +1,39 @@
-# massa-as-sdk
+# Massa-as-sdk
 
-Massa Assemblyscript SDK
+_Massa Assemblyscript SDK_
+
+Collection of tools, objects and functions, that are specific in usage only for Massa smart contracts in AssemblyScript 
+
```
Comment (Thykof) [inline README.md:6]: Text from the issue

```suggestion

SDK enables us to import object classes (such as address, storage objects) and use them without having to write all objects from scratch each time.

In order to call ABI functions, we import those functions in the smart-contract project, from the SDK dependency.
SDK is a set of tools, objects and functions, that are specific in usage only for Massa blockchain.

Contract standards (ERC-20, etc) have been removed from it.

```

**Review by Thykof — APPROVED** [review]

File: README.md [hunk 7]
```diff
@@ -1,3 +1,46 @@
...
+SDK enables us to import object classes (such as address, storage objects) and use them without having to write all objects from scratch each time.
+
+In order to call ABI functions, we import those functions in the smart-contract project, from the SDK dependency.
+SDK is a set of tools, objects and functions, that are specific in usage only for Massa blockchain.
```
[the remark this answers is not in the document]
Comment (longgamma-massa) [reply README.md:10]: it was a repetition from above

File: README.md [hunk 8]
```diff
@@ -1,3 +1,46 @@
...
+
+Contract standards (ERC-20, etc) have been removed from it.
```
Comment (longgamma-massa) [inline README.md:12]: I'd remove this. We don't need to say in the documentation that we removed some parts. We will mention such things in the releases.

File: README.md [hunk 9]
```diff
@@ -1,3 +1,46 @@
...
+
+Complete doc of all functions and object can be found here :
```
Comment (longgamma-massa) [inline README.md:14]:
```suggestion
Complete documentation of all available functions and objects is here: 
```

File: README.md [hunk 10]
```diff
@@ -1,3 +1,46 @@
...
+```
+
+## To generate doc
+```sh
+npm run doc
+```
+
+# Example
+
+For instance here is a smart contract example generating an event "Hello World" on the blockchain
```
Comment (longgamma-massa) [inline README.md:39]:
```suggestion
Here is a smart-contract example generating an event "Hello World" on the blockchain:
```

**Review by longgamma-massa — APPROVED** [review]

**Review by Thykof — APPROVED** [review]

**Pull request merged**